src/clients/c++/perf_client/perf_client.cc

```diff
@@ -118,144 +119,32 @@ SignalHandler(int signum)
 //     of "throughput, latency, concurrent request count" tuples will be
 //     reported in increasing load level order.
 //
+// - Maintaining Target Request Rate:
+//     Unlike above, here the client will maintain a fixed rate of requests
+//     issued to the server (see --request-rate-range option) while taking
+//     measurements. See RequestRateManager for more detail. Rest of the
```

Comment:
Shouldn't ask them to look in the code for more information. If something is important should say it here.